test/ok_test.exs

```diff
@@ -44,36 +44,57 @@ defmodule OKTest do
     assert {:error, :some_reason} = OK.failure(:some_reason)
   end
 
-  test "macro passes success value to function" do
-    report_func = fn (arg) -> send(self, arg) end
+  # test "macro passes success value to function" do
```

Comment:
Are the `ok_test.exs` comments and old tests still viable? You can always tag the current commit hash before merging the PR to refer back to the code if you need/want it.